explore/src/main/scala/explore/tabs/TargetTabContents.scala

```diff
@@ -391,7 +394,6 @@ object TargetTabContents extends TwoPanels:
                 props.userId,
                 targetId.some,
                 none,
-                // TODO PM correct the coordinates
                 CoordinatesAtVizTime(Target.Sidereal.baseCoordinates.get(target.get)).some,
```

Comment:
Are these PM corrected?

Reply:
There is no time in this case, we can only fallback to base